src/main/java/core/basesyntax/service/AuthorizationService.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exception.WrongLoginOrPasswordException;
+import core.basesyntax.model.User;
+
+public interface AuthorizationService {
+    User authorization(String login, String password) throws WrongLoginOrPasswordException;
```

Comment:
```suggestion
    User authorize(String login, String password) throws WrongLoginOrPasswordException;
```
method names should be verbs